src/test/java/com/puppycrawl/tools/checkstyle/bdd/InlineConfigParser.java

```diff
@@ -68,14 +64,8 @@ private static String getPackageFromFilePath(String filePath) {
         return path.substring(beginIndex, endIndex);
     }
 
-    private static List<String> readFile(Path filePath) throws CheckstyleException {
-        try (BufferedReader br = Files.newBufferedReader(
-                filePath, StandardCharsets.UTF_8)) {
-            return new ArrayList<>(br.lines().collect(Collectors.toList()));
-        }
-        catch (IOException ex) {
-            throw new CheckstyleException("Failed to read " + filePath, ex);
-        }
+    private static List<String> readFile(Path filePath) throws IOException {
+        return Files.readAllLines(filePath);
```

Comment:
you have changed method signature, which is not a good idea in general.
Before it was throwing CheckstyleException insted of IOException, please leave it as before and only fix the convertion to list part.

Reply:
Please don't apologize. I've pushed changes.